tests/functional/test_libraries.py

```diff
@@ -70,10 +67,6 @@ def test_PyQt4_QtWebKit(pyi_builder):
 @pytest.mark.xfail(reason='Reports "ImportError: No module named QtWebKit.QWebView".')
 @importorskip('PyQt4')
 def test_PyQt4_uic(tmpdir, pyi_builder, data_dir):
-    # Copy the data/PyQt4-uic.ui file to the tempdir used by this test.
-    os.mkdir(os.path.join(tmpdir.strpath, 'data'))
-    shutil.copy(os.path.join(data_dir, 'PyQt4-uic.ui'),
-                os.path.join(tmpdir.strpath, 'data'))
-
+    # Note that including the data_dir fixture copies files needed by this test.
```

Comment:
I think this is a good point: py.test relies on a lot of magic, and a newcomer to the test framework will be quite puzzled. I was thinking of putting some text in `tests/README.txt` -- I can create a separate pull request and would appreciate feedback / help in covering all the bases.